packages/hardhat-plugin/src/index.ts

```diff
@@ -318,15 +318,12 @@ ignitionScope
 
 ignitionScope
   .task("wipe")
-  .addParam("id", "The id of the deployment that has the future to wipe")
-  .addParam("future", "The id of the future to wipe")
+  .addParam("deploymentId", "The id of the deployment with the future to wipe")
```

Comment:
Should these be positional too?

Reply:
I have switched both `deploymentId` and `futureId` to be positional.